xmtp_mls/src/storage/sql_key_store.rs

```diff
@@ -41,11 +33,11 @@ impl OpenMlsKeyStore for SqlKeyStore<'_> {
     ///
     /// Returns an error if storing fails.
     fn store<V: MlsEntity>(&self, k: &[u8], v: &V) -> Result<(), Self::Error> {
-        let entry = StoredKeyStoreEntry {
-            key_bytes: k.to_vec(),
-            value_bytes: db_serialize(v)?,
-        };
-        entry.store(&mut self.store.conn()?)?;
+        self.store.insert_or_update_key_store_entry(
```

Comment:
This does an INSERT OR REPLACE instead of just a plain INSERT, so that we can overwrite existing values. I ran into this when trying to save the updated group state.